jerry-core/ecma/operations/ecma-jobqueue.c

```diff
@@ -346,6 +346,23 @@ ecma_process_all_enqueued_jobs (void)
   return ret;
 } /* ecma_process_all_enqueued_jobs */
 
+/**
+ * Release enqueued Promise jobs.
+ */
+void
+ecma_free_all_enqueued_jobs (void)
+{
+  while (JERRY_CONTEXT (job_queue_head_p) != NULL)
+  {
+    ecma_job_queueitem_t *item_p = JERRY_CONTEXT (job_queue_head_p);
+    JERRY_CONTEXT (job_queue_head_p) = JERRY_CONTEXT (job_queue_head_p)->next_p;
```

Comment:
`item_p->next_p` instead of `JERRY_CONTEXT (job_queue_head_p)->next_p` ?

Reply:
True, that's simpler to write. Thank you!